src/game/object/area.cpp

```diff
@@ -150,6 +150,13 @@ void Area::loadCameraStyle(const GffStruct &are) {
     _cameraStyle.pitch = styleTable->getFloat(styleIdx, "pitch", 0.0f);
     _cameraStyle.viewAngle = styleTable->getFloat(styleIdx, "viewangle", 0.0f);
     _cameraStyle.height = styleTable->getFloat(styleIdx, "height", 0.0f);
+
+    auto combatStyleRow = styleTable->findRowByColumnValue("name", "Combat");
```

Comment:
Loading camera style from a 2DA row could be extracted to a separate function, e.g. `void CameraStyle::load(const TwoDaRow &);`.

Reply:
`CameraStyle` is in `type.h`, shall I add a `types.cpp` for this?

Reply:
I'd add camerastyle.(h,cpp) files. Types were intended to store constants and enums and interfaces.